app/controllers/predicates/AuthPredicate.scala

```diff
@@ -107,27 +107,27 @@ class AuthPredicate @Inject()(authService: EnrolmentsAuthService,
       case Right(details) =>
         (details.isInsolventWithoutAccess, details.insolvencyDateFutureUserBlocked(dateService.now())) match {
           case (true, futureDateBlock) =>
-            logger.debug("[AuthPredicate][insolvencySubscriptionCall] - User is insolvent and not continuing to trade")
+            debug("[AuthPredicate][insolvencySubscriptionCall] - User is insolvent and not continuing to trade")
             Future.successful(
               Forbidden(userInsolventError()(user, request2Messages, appConfig)).addingToSession(
                 SessionKeys.insolventWithoutAccessKey -> "true",
                 SessionKeys.futureInsolvencyBlock -> s"$futureDateBlock")
             )
           case (_, true) =>
-            logger.debug("[AuthPredicate][insolvencySubscriptionCall] - User has a future insolvencyDate, throwing ISE")
```

Comment:
same here

Reply:
i think this can be seen an a validation error and therefore changed to warnLog